### PR TITLE
Add setup.py

### PR DESCRIPTION
**Pull request by damonkelley**
1. Adds `setup.py`
2. Fixes some import statements
3. Removes an unused import from an external package
4. Reorganize the metadata

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    version=version,
+    packages=find_packages(),
+    include_package_data=True,
+    license='BSD',
+    description='A helper library for Coda.',
+    long_description=readme,
+    keywords='library anvl coda',
+    author='University of North Texas Libraries',
+    author_email='mark.phillips@unt.edu',
+    url='https://github.com/unt-libraries/codalib',
```
Comment (ldko) [inline setup.py:31]: Let's put a slash on the end of this URL.

File: codalib/util.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 from lxml import etree
 import uuid
 import datetime
-from coda import bagatom
-import httplib2
```
Comment (ldko) [inline codalib/util.py:15]: If we are taking out this import, let's delete this commented out use:
https://github.com/unt-libraries/codalib/blob/add-setup/codalib/util.py#L65

File: setup.py (hunk 1 above)
Comment (damonkelley) [inline setup.py:31]: FWIW, GitHub does not use trailing slashes, and will strip them off if they are included.

File: codalib/util.py (hunk 2 above)
Comment (damonkelley) [inline codalib/util.py:15]: How do you feel about removing all of those commented out lines that refer to the the `httplib` object?

File: setup.py (hunk 1 above)
Comment (ldko) [inline setup.py:31]: WHAT!? Nevermind then!

File: codalib/util.py (hunk 2 above)
Comment (ldko) [inline codalib/util.py:15]: Yes, please do that.

**Comment by ldko** [comment]
:+1: Goodbye loop for-evah

**Pull request merged**